src/internal/handle_file_operation_test.go

```
@@ -165,3 +166,316 @@ func TestCompressSelectedFiles(t *testing.T) {
+
+			if tt.shouldPreventPaste {
+				// Verify that paste was prevented (original should still exist, clipboard not cleared)
+				if originalPath != "" {
+					if strings.Contains(originalPath, ".txt") {
+						assert.FileExists(t, originalPath, "Original file should still exist when paste is prevented")
+					} else {
+						assert.DirExists(t, originalPath, "Original directory should still exist when paste is prevented")
+					}
+				}
+				// Clipboard should not be cleared when paste is prevented
+				assert.NotEqual(t, 0, len(m.copyItems.items), "Clipboard should not be cleared when paste is prevented")
```

golangci-lint (1.64.8)

empty: use assert.NotEmpty

(testifylint)